### DISCUSSION TITLE
no packages available

### DISCUSSION THREAD

**Discussion by jekv2** [FossifyOrg/General-Discussion#74] [Q&A]
no paxkages available for fossify apps.[
![Screenshot_20240107-152443_1](https://github.com/FossifyOrg/General-Discussion/assets/7735249/1beb9e72-a0f4-42d5-9012-c0e82a896479)
](url)

**Comment by Aga-C** [comment]
Apps are being gradually released on GitHub and F-Droid. Currently, there are Gallery, Calendar and File Manager, but the rest will also be released.
